run_scans.py

```diff
@@ -107,10 +107,7 @@ def launchTestsArgs(tool, shelf, slot, link, chamber, scanmin, scanmax, stepSize
     cmd.append( "--scanmax=%d"%(scanmax) )
     cmd.append( "--nevts=%d"%(nevts) )
     cmd.append( "--throttle=%i"%(throttle) )
-    if stepSize > 0:
-      step = stepSize
-      if (step + scanmin > scanmax):
-        step = scanmax - scanmin
+    if stepSize:
       cmd.append( "--stepSize=%d"%(step) )
```

Comment:
Ack, I did have options.stepSize there, but b/c of the git push CLI issues, I updated to code on the web GUI, and missed that one.